diff --git a/c_api/CMakeLists.txt b/c_api/CMakeLists.txt
index 01789a32..79892046 100644
--- a/c_api/CMakeLists.txt
+++ b/c_api/CMakeLists.txt
@@ -50,6 +50,13 @@ file(GLOB FAISS_C_API_HEADERS
      "utils/*.h")

 faiss_install_headers("${FAISS_C_API_HEADERS}" c_api)
+install(
+  TARGETS faiss_c
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+)

 add_executable(example_c EXCLUDE_FROM_ALL example_c.c)
 target_link_libraries(example_c PRIVATE faiss_c)
